plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -777,12 +802,90 @@ public static function get_plugin( $slug ) {
 				'manage_url'     => 'admin.php?page=tiktok',
 				'is_built_by_wc' => false,
 				'is_visible'     => false,
-			],
+			),
 		);
 
 		$plugin        = $plugins[ $slug ];
 		$plugin['key'] = $slug;
 
 		return $plugin;
 	}
+
+	public static function withCoreProfilerFields( $plugins ) {
```

Comment:
Some linting error here. 🥲 

```
FOUND 3 ERRORS AND 0 WARNINGS AFFECTING 3 LINES
-----------------------------------------------------------------------------------------------
 814 | ERROR | Method name "withCoreProfilerFields" in class DefaultFreeExtensions is not in snake case format, try "with_core_profiler_fields" (WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid)
 814 | ERROR | Missing doc comment for function withCoreProfilerFields() (Squiz.Commenting.FunctionComment.Missing)
 869 | ERROR | Use Yoda Condition checks, you must. (WordPress.PHP.YodaConditions.NotYoda)
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/38418/commits/382d253623957a552e3cf21f09e5b6464365700b